js/screens/Activity/styles.js

```diff
@@ -1,2 +1,35 @@
-import { StyleSheet } from "react-native";
-export default (style = StyleSheet.create({}));
+import { StyleSheet, Dimensions } from "react-native";
+const styles = StyleSheet.create({
+  background: {
+    width: Dimensions.get("window").width,
+    height: Dimensions.get("window").height
+  },
+  container: {
+    flex: 1,
+    justifyContent: "space-evenly",
+    height: Dimensions.get("window").height * 1.02
```

Comment:
deconstruct height and width on the top so you can easily just type height and width without huge syntax.